src/main.rs

```diff
@@ -1,54 +1,9 @@
-use std::io::{Result, Error, ErrorKind};
-use std::sync::Arc;
-use std::sync::mpsc::{channel, Sender, Receiver};
-
```

Comment:
This `use` statement should not affect external consumers of `zecwalletlib`.

First, `use` by itself doesn't expose names externally, and second this is a different crate to the `zecwalletlitelib`.